hangman.py

```diff
@@ -0,0 +1,137 @@
+import random
+import os
+import pickle
+
+### FUNCTIONS ###
+
+
+def display_title():
+    os.system('cls')
+    print("################################")
+    print("#### You're playing hangman ####")
+    print("################################")
+
+
+def get_choice():
+    print("\nEnter [1] to play Hangman. ")
+    print("Enter [2] to add your word to the pool.")
+    print("Enter [3] to see pool of words.")
+    print("Enter [q] to quit game.")
+    choice = input("What would you like to do? ")
+    return choice
+
+
+def load_words():
+    # This function loads names from a file, and puts them in the list 'names'.
+    #  If the file doesn't exist, it creates an empty list.
+    try:
+        file_object = open('words.txt', 'rb')
+        names = pickle.load(file_object)
+        file_object.close()
+        return names
+    except Exception as e:
+        print(e)
+        return []
+
+
+def quit():
+    # This function dumps the names into a file, and prints a quit message.
+    try:
+        file_object = open('words.txt', 'wb')
+        pickle.dump(words, file_object)
+        file_object.close()
+        print("\nThanks for playing. I will remember these words.")
+    except Exception as e:
+        print("\nThanks for playing. I won't be able to remember these words.")
+        print(e)
+
+
+def hangman():
```

Comment:
`hangman` is a bit generic name, I would call it smth like `play` or `play_hangman` or `start_game`.